ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fragmentcontainer/v1/fragmentcontainer/_cq_design_dialog/.content.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If there are no changes from super resource type, you don't need to create, it will extend.